book_source/source/module-9-miscellaneous-topics/lesson-27-reading-ethics-in-data-science/case-study-1-tracking-for-safety.md

```diff
@@ -0,0 +1,45 @@
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Want to see the true potential impact of ignoring social distancing? Through a partnership with <a href="https://twitter.com/xmodesocial?ref_src=twsrc%5Etfw">@xmodesocial</a>, we analyzed secondary locations of anonymized mobile devices that were active at a single Ft. Lauderdale beach during spring break. This is where they went across the US: <a href="https://t.co/3A3ePn9Vin">pic.twitter.com/3A3ePn9Vin</a></p>&mdash; Tectonix (@TectonixGEO) <a data-dnt="true" href="https://twitter.com/TectonixGEO/status/1242628347034767361?ref_src=twsrc%5Etfw">March 25, 2020</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+When another user poited out a concern about a potential violation of the user's privacy (tweet since deleted), Textronix [replied](https://mobile.twitter.com/TectonixGEO/status/1243004792932368385).
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Understand the concern, but every point of data we used here is completely anonymized and collected with user consent! We realize the implications of data collection at this scale, but used responsibly with privacy in mind, it can have massive positive effect!</p>&mdash; Tectonix (@TectonixGEO) <a data-dnt="true" href="https://twitter.com/TectonixGEO/status/1243004792932368385?ref_src=twsrc%5Etfw">March 26, 2020</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+(content:module9:lesson27:case-study-1:things-to-consider)=
+
+## Things to Consider
+
+This case study sets up an interesting tension between utility (i.e. public safety) and user's privacy. On one hand, we want to have accurate information to inform policy-makers on the general behaviors of people so they can make better informed policy decisions. But in some cases, this utility can come at the cost of someone's personal privacy.
+
+Now, Tectronix claims (see tweet above) that there is no privacy concerns here since they collected the data with users' consent and kep the data anonymous. While we assume this is true, there are still a couple potential gray-areas here.
+
+1. **Anonymous data sometimes isn't**: In the [next module](/module-10-beyond-163/lesson-29-privacy/index), we'll discuss the challenges of keeping anonymous data truly anonymous. You'll find that it's surprisingly easy, given enough information, to narrow down a large dataset with anonymous data, down to a single individual that you are looking for. In that lesson, we will talk about some techniques to ensure anonimity, but the key point is that it's challenging and rarely sufficient to just remove the users' name from the data. You can also see an example linked in [](content:module9:lesson27:case-study-1:further-resources) below on how the New York Times could identify indiviudals from a large, anonymous dataset.
```

Comment:
Feels more natural to have something after "isn't".
"anonimity" -> "anonymity"
"indiviudals" -> "individuals"